lib/explorer/backend/lazy_series.ex

```diff
@@ -40,14 +40,20 @@ defmodule Explorer.Backend.LazySeries do
     median: 1,
     var: 1,
     std: 1,
-    quantile: 2
+    quantile: 2,
+    # Maybe aggregations
+    first: 1,
+    last: 1,
+    count: 1,
+    # Slice and dice
+    coalesce: 2
```

Comment:
```suggestion
    # Slice and dice
    coalesce: 2
```

I would move this to the before "Aggregations" and we will likely have a bunch more operations like this in the future. We will have starts_with, contains, ends_with and so on.